MoreLinq/TagFirstLast.cs

```diff
@@ -63,5 +63,34 @@ partial class MoreEnumerable
             return source.Index() // count-up
                          .CountDown(1, (e, cd) => resultSelector(e.Value, e.Key == 0, cd == 0));
         }
+
+        /// <summary>
+        /// Returns a sequence of tuples, where the N-th tuple contains the N-th
+        /// element of the source sequence and two booleans indicating whether the
+        /// element is the first and/or last.
+        /// </summary>
+        /// <typeparam name="TSource">The type of the elements of <paramref name="source"/>.</typeparam>
+        /// <param name="source">The source sequence.</param>
+        /// <returns>
+        /// Returns the resulting sequence.
+        /// </returns>
+        /// <remarks>
+        /// This operator uses deferred execution and streams its results.
+        /// </remarks>
+        /// <example>
+        /// <code><![CDATA[
+        /// var numbers = new[] { 123, 456, 789 };
+        /// var result = numbers.TagFirstLast();
+        /// ]]></code>
+        /// The <c>result</c> variable, when iterated over, will yield
+        /// <c>(Value = 123, IsFirst = <see langword="true"/>, IsLast = <see langword="false"/>)</c>,
```

Comment:
The `<see langword="..."/>` application is incorrect here as the output is literally the string `"True"` and not the keyword `true`, so this needs changing to:

```suggestion
        /// <c>(123, True, False)</c>, <c>(456, False, False)</c> and
```

Reply:
There is no other point in the documentation where the actual name of the ValueTuple fields are given. In this case it's obvious, but for other cases (`Lag` is a good example), we have to take care of this.

Reply:
See https://github.com/dotnet/csharplang/issues/145